DataManagementSystem/DB/FileCatalogComponents/SecurityManager.py

```diff
@@ -297,6 +299,14 @@ def __init__( self, database = False ):
     pluginCls = self.__loadPlugin( pluginPath )
     self.policyObj = pluginCls( database = database )
 
+    # For the old clients to work with the new policy (since getPathPermissions is meant to disappear...)
+    # we fetch the old SecurityManager, and we call it if needed in the plugin.
+    oldSecurityManagerName = gConfig.getValue( cfgPath( serviceSection, 'OldSecurityManager' ), '' )
+    self.policyObj.oldSecurityManager = None
+    if oldSecurityManagerName:
+      self.policyObj.oldSecurityManager = eval( "%s(self.db)" % oldSecurityManagerName )
```

Comment:
import ast
ast.literal_eval()

is advocated as a better, more secure solution instead of eval()